src/routes/namespaces/[namespace]/workflows/[workflow]/[run]/history/compact/__layout.svelte

```diff
@@ -2,26 +2,24 @@
   import type { LoadInput } from '@sveltejs/kit';
 
   export const load = async ({ stuff }: LoadInput) => {
-    const { events } = stuff;
-
-    console.log({ events });
+    const { events } = stuff as { events: HistoryEventWithId[] };
```

Comment:
Man shoving stuff into stuff seems to be potentially difficult to type. It'd be nice if we didn't have to as this and we had some kind of global stuff type like we did with getContext but not super sure how to implement this. We definitely need to parse what is in stuff as a human so it's hard to involve the compiler on this.

I wonder if there's a empty/404 case we're missing here.